packages/neo-one-node/src/FullNode.ts

```diff
@@ -34,7 +38,23 @@ export class FullNode {
             reject(error);
           }
         };
-        this.mutableSubscription = fullNode$(this.options).subscribe({
+        this.mutableSubscription = combineLatest(
```

Comment:
I'm thinking `merge` is more applicable here?

Reply:
I would just subscribe to the `interval` observable separately - `start` should resolve once the node has started, and trying to combine them seems problematic.